src/restic/node.go

```diff
@@ -331,11 +321,26 @@ func (node *Node) UnmarshalJSON(data []byte) error {
 	type nodeJSON Node
 	nj := (*nodeJSON)(node)
 
-	err := json.Unmarshal(data, nj)
-	if err != nil {
-		return errors.Wrap(err, "Unmarshal")
+	if err := json.Unmarshal(data, nj); err != nil {
+		// Retry conversion with list of ExtendedAttributes
```

Comment:
That's done in integration testing, right? if it can still read small-repo.tar.gz and consorts, the legacy path is fine.

Also, from a bigger picture, isn't this the best way forward? How else would you make json effecting changes?